pvlib/iotools/psm4.py

```diff
@@ -314,9 +314,6 @@ def get_nsrdb_psm4_tmy(latitude, longitude, api_key, email, year='tmy',
     latitude = ('%8.4f' % latitude).strip()
     # TODO: make format_WKT(object_type, *args) in tools.py
 
-    # convert to string to accomodate integer years being passed in
-    year = str(year)
-
```

Comment:
Why is this only removed for one of the functions? I see this is present for the other three functions as well.

I support the removal since this line doesn't seem to be necessary, but we should be consistent.

Reply:
Exactly! I just ran the tests locally with all the ones removed all tests pass.